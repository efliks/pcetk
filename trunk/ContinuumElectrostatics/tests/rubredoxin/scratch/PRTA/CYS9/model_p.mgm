ON_GEOM_CENT 121 2.000000
(11.488790 48.768688 4.370910) 101 1.000000
(11.488790 48.768688 4.370910) 101 0.500000
(11.488790 48.768688 4.370910) 101 0.250000
