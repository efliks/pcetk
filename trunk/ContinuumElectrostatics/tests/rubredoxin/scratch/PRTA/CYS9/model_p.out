Starting /home/mikolaj/local/bin//my_2diel_solver for molecule named scratch/PRTA/CYS9//site_p
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
/home/mikolaj/local/bin//my_2diel_solver: WARNING:  The atom, "0:0:0:0:9:CB",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
/home/mikolaj/local/bin//my_2diel_solver: WARNING:  The atom, "0:0:0:0:9:HB1",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
/home/mikolaj/local/bin//my_2diel_solver: WARNING:  The atom, "0:0:0:0:9:HB2",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
/home/mikolaj/local/bin//my_2diel_solver: WARNING:  The atom, "0:0:0:0:9:HG1",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
/home/mikolaj/local/bin//my_2diel_solver: WARNING:  The atom, "0:0:0:0:9:SG",
   occurs in both the solute and protein atom sets.
   For defining the Electrolyte boundary, the protein version
   will be used.
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (13.0093, 48.7291, 2.52728)) 121 2
Finer level:    (11.4888, 48.7687, 4.37091) 101 1
Finer level:    (11.4888, 48.7687, 4.37091) 101 0.5
Finest level:   (11.4888, 48.7687, 4.37091) 101 0.25


Self energy of scratch/PRTA/CYS9//site_p in two dielectric environment = 26.1151
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of scratch/PRTA/CYS9//site_p with background charge set in two dielectric environment = -0.248102
(probably in kcal/mole)
Total runtime: 13.96 s user 0.13 s system 
